src/components/Person.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const misha = {
+  name: 'Misha',
+  age: 37,
+  sex: 'm',
+  isMarried: true,
+  partnerName: 'Natasha',
+};
+
+// eslint-disable-next-line
+export const olya = {
+  name: 'Olya',
+  sex: 'f',
+  isMarried: true,
+  partnerName: 'Maksym',
+};
+
+// eslint-disable-next-line
+export const alex = {
+  name: 'Alex',
+  age: 25,
+  sex: 'm',
+  isMarried: false,
+};
+
+export const Person = ({ person }) => {
+  const {
+    name, age, sex, isMarried, partnerName,
+  } = person;
+
+  let partner = 'husband';
+
+  if (sex === 'm') {
+    partner = 'wife';
+  }
+
+  return (
+
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      <p className="Person__age">
+        { age
+          ? `I am ${age}`
+          : null}
+      </p>
```

Comment:
Don't render this paragraph if there is no age
```suggestion

        { age
          ? <p> `I am ${age}` </p>
          : null}
```